File: 36-team-management/npm-package/teams.ts
import axios from 'axios';
const API = 'https://cost-katana-backend.store/api'\;
const KEY = process.env.COST_KATANA_API_KEY;

export async function createTeam(name: string) {
  const res = await axios.post(`${API}/teams`, {name},
    {headers: {'Authorization': `Bearer ${KEY}`}});
  console.log('✅ Team created:', res.data.data.id);
  return res.data.data;
}

export async function addMember(teamId: string, email: string, role: string) {
  const res = await axios.post(`${API}/teams/${teamId}/members`, {email, role},
    {headers: {'Authorization': `Bearer ${KEY}`}});
  console.log('✅ Member added:', email);
  return res.data.data;
}

File: 35-team-management/python-sdk/teams.py
import os, requests
API = "https://api.costkatana.com/api"
KEY = os.getenv("COST_KATANA_API_KEY")

def create_team(name):
    res = requests.post(f"{API}/teams",
        json={"name": name},
        headers={"Authorization": f"Bearer {KEY}"})
    res.raise_for_status()
    print(f"✅ Team created: {res.json()['data']['id']}")
    return res.json()["data"]

def add_member(team_id, email, role):
    res = requests.post(f"{API}/teams/{team_id}/members",
        json={"email": email, "role": role},
        headers={"Authorization": f"Bearer {KEY}"})
    res.raise_for_status()
    print(f"✅ Member added: {email}")
    return res.json()["data"]
